nan.h

```diff
@@ -2189,6 +2189,42 @@ inline void SetCallAsFunctionHandler(
 
 #include "nan_object_wrap.h"  // NOLINT(build/include)
 
+//=== HiddenValue/Private ======================================================
+
+v8::Local<v8::Value> GetPrivate(
+    v8::Local<v8::Object> object,
+    v8::Local<v8::String> key) {
+#if (NODE_MODULE_VERSION > NODE_6_0_MODULE_VERSION)
+  v8::Isolate* isolate = v8::Isolate::GetCurrent();
+  v8::Local<v8::Context> context = isolate->GetCurrentContext();
+  v8::Local<v8::Private> privateKey = v8::Private::ForApi(isolate, key);
```

Comment:
Minor style nit but I think nan predominantly uses snake_case for locals, not camelCase.